mne/decoding/tests/test_time_gen.py

```diff
@@ -70,7 +70,15 @@ def test_generalization_across_time():
     assert_equal("<GAT | fitted, start : -0.200 (s), stop : 0.499 (s), no "
                  "prediction, no score>", '%s' % gat)
     assert_equal(gat.ch_names, epochs.ch_names)
+    # test different predict function:
+    gat.decision_function(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 1))
+    gat.predict_proba(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 2))
```

Comment:
What do you suggest instead ?

On Sunday, 3 January 2016, Alexandre Gramfort <notifications@github.com>
wrote:

> In mne/decoding/tests/test_time_gen.py
> <https://github.com/mne-tools/mne-python/pull/2755#discussion_r48695241>:
>
> > @@ -70,7 +70,15 @@ def test_generalization_across_time():
> >      assert_equal("<GAT | fitted, start : -0.200 (s), stop : 0.499 (s), no "
> >                   "prediction, no score>", '%s' % gat)
> >      assert_equal(gat.ch_names, epochs.ch_names)
> > +    # test different predict function:
> > +    gat.decision_function(epochs)
> > +    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 1))
> > +    gat.predict_proba(epochs)
> > +    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 2))
>
> so now y_pred_ contains proba? I don't like that... that feels broken
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2755/files#r48695241>.
>